src/main/resources/additional_properties.json

```diff
@@ -18,5 +18,266 @@
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "whitelist" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                },
+                "blacklist" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                },
+                "renames" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "set_schema_metadata" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "set_schema_metadata" : {
+              "additionalProperties" : false,
+              "required" : [ "apply_to", "name", "version" ],
+              "properties" : {
+                "apply_to" : {
+                  "$ref" : "#/$defs/apply_to"
+                },
+                "name" : {
+                  "type" : "string",
+                  "default" : null
+                },
+                "version" : {
+                  "type" : "integer",
+                  "default" : null
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "tombstone_handling" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "tombstone_handling" : {
+              "additionalProperties" : false,
+              "required" : [ "behaviour" ],
+              "properties" : {
+                "behaviour" : {
+                  "type" : "string",
+                  "default" : "warn",
+                  "enum" : [ "warn", "ignore", "fail" ]
+                }
+              }
+            }
+          }
+        },
+        {
+          "type" : "object",
+          "required" : [ "value_to_key" ],
+          "additionalProperties" : false,
+          "properties" : {
+            "value_to_key" : {
+              "additionalProperties" : false,
+              "required" : [ "fields" ],
+              "properties" : {
+                "fields" : {
+                  "type" : "array",
+                  "items" : {
+                    "type" : "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "type": "object",
+          "required": [ "topic_routing" ],
+          "additionalProperties": false,
+          "properties": {
+            "topic_routing": {
+              "additionalProperties": false,
+              "required": [ "topic_regex", "topic_replacement", "key_enforce_uniqueness", "key_field_name", "key_field_regex", "key_field_replacement" ],
```

Comment:
`key_enforce_uniqueness` and `key_field_name` have defaults and aren't mandatory fields. You  can add the defaults to the schema too.